### PR TITLE
Change SERVICENOW_ID references to ASSESSMENT_ID

### LINKED ISSUE #30: Remove 3rd party reference from arguments
# 💡 Summary #

A current argument includes the name of a 3rd party dependency ("ServiceNow ID" ). Ideally we would want to change this reference to something more generic.

## Motivation and context ##

Currently the TPT service is in development, and the definition of this ID could use some clarification. Will it be a "Customer ID" or a "Job Ticket ID" or combination of both over time. Original planning of the TPT service allows for multiple "Test" which would produce multiple reports over a given period of time. 

Converting this to a generic term would improve the 'adaptability' of this repo.

## Implementation notes ##

- Lockdown definition of the "ID" used
- Update codebase to support a more generic term once defined.

## Acceptance criteria ##

- [ ] Application Usage has no 3rd party references

### PR DESCRIPTION
**Pull request by nickviola**
#  #

## 🗣 Description ##
This PR resolves #30 and changes all references of SERVICENOW_ID to ASSESSMENT_ID in this project.

## 💭 Motivation and context ##
Leadership has requested these changes be implemented to clarify ID references.

## 🧪 Testing ##
Run the script as usage suggests and verify output references are replaced.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ The package collects raw data and creates an encrypted PDF.
...
+  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
 
 Options:
   -h --help                         Show this message.
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
 Arguments:
-  SERVICENOW_ID                     The ID number in Service Now
+  ASSESSMENT_ID                     The assigned Assessment ID.
```
Comment (dav3r) [inline README.md]: May I suggest a slight rewording here:
```suggestion
  ASSESSMENT_ID                     The assessment identifier.
```

File: src/tpt_reports/tpt_reports.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
 
 Options:
   -h --help                         Show this message.
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
 Arguments:
-  SERVICENOW_ID                     The ID number in Service Now
+  ASSESSMENT_ID                     The assigned Assessment ID.
```
Comment (dav3r) [inline src/tpt_reports/tpt_reports.py]: May I suggest a slight rewording here:
```suggestion
  ASSESSMENT_ID                     The assessment identifier.
```

File: src/tpt_reports/tpt_reports.py [hunk 3]
```diff
@@ -141,10 +141,10 @@ def main() -> None:
                 + "debug, info, warning, error, and critical.",
             ),
             # Issue #30 - Remove 3rd party reference from arguments
-            # TODO: Define a generic ID format to replace SERVICENOW_ID and provide validation.
+            # TODO: Define a generic ID format to replace ASSESSMENT_ID and provide validation.
```
Comment (dav3r) [inline src/tpt_reports/tpt_reports.py]: These two comment lines can be deleted since this PR resolves #30.

**Review by dav3r — CHANGES_REQUESTED** [review]
In addition to the small things I noted, your PR description should also mention that this PR resolves #30.

File: src/tpt_reports/tpt_reports.py (hunk 3 above)
Comment (nickviola) [reply src/tpt_reports/tpt_reports.py]: Resolved in 7e32d31

**Review by dav3r — APPROVED** [review]
👍 👍

**Review by schmelz21 — APPROVED** [review]
looks good to me!

**Pull request merged**